### DISCUSSION TITLE
Revert action of output_notebook()

### DISCUSSION THREAD

**Discussion by DrNickBailey** [bokeh/bokeh#13580] [Internals and design (Q&A)]
How do you disable the action of output_notebook()? I feel it should have a parameter to disable it, otherwise it 'steals' the whole notebook's ability to output to the browser as it did by default without the ability to revert.

**Comment by bryevdv** [comment]
I've never seen anything like this at all, I don't know what referring to.  Please describe in much more detail, full system and package version information, screenshots and/or video, and provide a minimal reproducing example.

**Comment by DrNickBailey** [comment]
This is the documentation for `output_notebook()`, it's part of bokeh.io and comes up in google/stack overflow as the means to get bokeh plots to show in the JupyterLab notebook rather than loading in a separate browser tab:
https://docs.bokeh.org/en/latest/docs/reference/io.html#bokeh.io.output_notebook

But there seems to be no way to revert this behaviour once `output_notebook()` is first called. This seems like an oversight. If there is a switch it is usually possible to turn the switch off (what I mean by "revert").

**Reply by bryevdv** [reply]
[`reset_output`](https://docs.bokeh.org/en/latest/docs/reference/io.html#bokeh.io.reset_output)
